config/database.yml

```diff
@@ -8,10 +8,14 @@ default: &default
 
 development:
   <<: *default
+  host: db
+  username: postgres
```

Comment:
@AllanKlaus could you please revert the changes to this file? I don't need them (nor does Travis CI), as far as I'm using trust policy for the local connections in the PostgreSQL configuration file, like this:
```
# TYPE  DATABASE        USER            ADDRESS                 METHOD
# "local" is for Unix domain socket connections only
local   all             all                                     trust
```